react/features/conference/components/web/Conference.js

```diff
@@ -146,8 +146,13 @@ class Conference extends AbstractConference<Props, *> {
      */
     componentDidMount() {
         document.title = `${this.props._roomName} | ${interfaceConfig.APP_NAME}`;
+        const { DISABLE_SUPERHERO = false } = interfaceConfig;
         const { address: addressParam, signature: signatureParam } = parseURLParams(window.location, true, 'search');
 
+        if (DISABLE_SUPERHERO === true) {
+            return this._start();
```

Comment:
@Chima1707 not sure what you mean. You want to stop function after that condition and run it?
And we also still have at the same file 
https://github.com/aeternity/jitsi-meet/blob/f86fb4a3f5f2c90dd68a8b4e75c077743c972a7b/react/features/conference/components/web/Conference.js#L180

So if that will stops we don't run `initClient()`. Not sure. Correct me if I'm wrong.